builtin/credential/chef-node/path_config.go

```diff
@@ -0,0 +1,75 @@
+package chefNode
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config",
```

Comment:
In the code I only see role/tag/environments; what would config/node (or just node) be for? If it's per-node policy bindings it should stay out of config/.